test/integration/patients/worklist/worklist.js

```diff
@@ -3740,7 +3741,8 @@ context('worklist page', function() {
 
     cy
       .get('@listSearch')
-      .should('have.attr', 'value', 'Nurse');
+      .invoke('val')
```

Comment:
the input didn't need to be re-rendered on restart, so the `value` attribute wasn't being re-rendered, so we have to read the input value instead.